src/test/scala/firrtlTests/ParserSpec.scala

```diff
@@ -92,8 +92,8 @@ class ParserSpec extends FirrtlFlatSpec {
     ) ++ PrimOps.listing
   }
 
-  // FIRRTL version number
-  "Version" should "be parsed" in {
+  // ********** FIRRTL version number **********
```

Comment:
should we also have an explicit test for "a file without a version number should be accepted"

Reply:
Technically every other test in this file is testing that but I do agree that calling it out explicitly would be useful